packages/zero-server/src/push-processor.ts

```diff
@@ -204,7 +205,7 @@ export class PushProcessor<
     params: Params,
     req: PushBody,
     m: Mutation,
-    errorMode: boolean,
+    caughtError: unknown,
   ): Promise<MutationResponse> {
```

Comment:
Should this still return a MutationResponse given that is now communicated via upstream?

Reply:
Yeah, we still need to check if the mutation response contains an `ooo mutation` error as that will not be written to upstream in the current design.